dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -31,12 +30,29 @@ app.use(cors());
 app.use(morgan("dev"));
 
 
-//health check
 app.get("/ping", (req, res) => {
   return res.status(200).json({ message: "pong" });
 });
 
 
+app.post("/users", async (req, res, next) => {
+  const { name, age, email, password } = req.body
+
+  await myDataSource.query(
```

Comment:
```suggestion
  await appDataSource.query(
```
위에서 Datasource 인스턴스를 appDataSource로 바꾸어 주었기 때문에 해당 코드도 수정이 필요합니다.
이대로 실행시켜 `/users` 엔드포인트를 실행하면 에러가 나게 됩니다.
마지막으로 수정 부탁드리겠습니다 :)

Reply:
아까 'Modify: 코드 수정'으로 커밋한 파일에 반영했는데 혹시 확인 안되실까요?